client/state/sites/blog-stickers/reducer.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { merge } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { SITES_BLOG_STICKER_LIST_RECEIVE } from 'state/action-types';
+import { combineReducers, createReducer } from 'state/utils';
+
+export const items = createReducer(
+	{},
+	{
+		[ SITES_BLOG_STICKER_LIST_RECEIVE ]: ( state, action ) => {
+			return merge( {}, state, {
+				[ action.payload.blogId ]: action.payload.stickers,
```

Comment:
keyed reducer requires you to have `blogId` as a top level key of the action (until we modify that aspect of it).  So would require you to change a couple things if you wanted to use it.  Would look something like:
```js
keyedReducer( 'blogId', createReducer( null, { 
   [ SITES_BLOG_STICKER_LIST_RECEIVE ]: ( state, action ) => action.payload
} )
```